front/src/layouts/footer/footer.css

```diff
@@ -0,0 +1,74 @@
+div.body {
+  color: white;
+  width: 1440px;
+}
+
+div.main, div.lowerPart {  
+  display: flex;
+  flex-direction: row;                        
+  align-items: center;
+  justify-content: space-between; 
+}
+
+div.main {
+  height: 125px;
+}
+
+div.logo {
+  margin: 5px 0px 10px 80px;
+  padding: 5px 0px 10px;   
+}
+
+div.stores {
+  margin-left: 80px;   
+}
+
+div.lowerPart {
+  height: 57px; 
+}
+
+button.privicyPolice {
+  margin-right: 140px;
+  font-size: 14px;
+}
+
+button {
+  color: white;
+  border: 0px;
+  padding: 10px;
+  font-size: 16px;
+}
+
+button:hover {
+  cursor: pointer;
+}
+
+div.socialNetworks {
+  margin-right: 115px; 
+}
+
+i {
+  font-size: 22px;
+  padding: 10px;
+}
+
+span.allReles {
```

Comment:
not specify div, span or other tag`s name if it hasn`t highly important difference